sections/participants.tex

```diff
@@ -1,57 +1,27 @@
+Transparency is necessary for applicants to understand acceptance/rejection decisions, and accountability is of crucial importance for the detection of inherent biases in the selection, which may harm both the event's success as well as the larger community.
+One way to maximize transparency in the selection process is to remove the human element from the decision making as far as possible and transfer some of these tasks to a well-designed algorithm.
+In this context, well-designed means \cite{o2017weapons}: (1) interpretable, i.e. the algorithm, its parameters and outputs can be understood by humans, (2) openly available, and thus the decision process can be inspected, (3) the algorithm itself does not perpetuate intrinsic biases in the data (though we will show below that an algorithm itself does not entirely remove the requirement of humans to be aware of their biases).
+In practice, most selection procedures follow two separate steps. In the first step, the merit of a candidate must be assessed: does the candidate fulfill the requirements posed by the goals of the workshop?
 This first step is difficult to automate, because it requires judgment calls, often based on long-form answers.
-In the past, we have performed this step by blinding ourselves to a candidate's other attributes (including name and other personal information), and assess their candidacy based solely upon key questions asked specifically for this purpose.
+To avoid implicit/unconcsious bias, in the past, we have sometimes performed this step by blinding ourselves to a candidate's other attributes (including name and other personal information), and assess their candidacy based solely upon key questions asked specifically for this purpose.
 When doing this procedure for a large enough sample, it is unlikely that the resulting pool of acceptable candidates is smaller than the number of available spaces at the workshop.
-The second step in the selection procedure thus requires tie-breaking between equally acceptable candidates.
+The second step in the selection procedure then requires tie-breaking between equally acceptable candidates.
 It is here where one may impose outside constraints on the selection based on the goals of the workshop.
 If multiple competing constraints are considered, this task essentially becomes a complex optimization problem, for which algorithms exist that will outperform any human selection procedure.
 
-In particular, the software \textit{entrofy}\footnote{\url{http://github.com/dhuppenkothen/entrofy}} employs one such algorithm and is capable of efficient selection of an optimal candidate set based on human target fractions.
+On solution to this optimization procedure is implemented in the software \textit{entrofy}\footnote{\url{http://github.com/dhuppenkothen/entrofy}}. The algorithm maximizes the variance of participants on specified dimensions (e.g., career stage, geographic location, etc.) to meet pre-set fractions set by the organizers.
```

Comment:
Addressed in https://github.com/uwescience/HackWeek-Writeup/pull/61/commits/efbdc725cf468830483e7514835ee29ca11157fe ?